CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 6.15.2 (September 27, 2022). Tested on Artifactory 7.41.13
```

Comment:
We should bump to 6.16.0 because we are adding new resource.

Reply:
ok